### DISCUSSION TITLE
Link Preview for ants 🐜(is too small to read)

### DISCUSSION THREAD

**Discussion by romeogiggletoes** [Chatterino/chatterino2#3462] [Ideas]
### Checklist

- [X] I'm reporting a problem with Chatterino
- [X] I've verified that I'm running **the most** recent nightly build or stable release
- [X] I've looked for my problem on the [wiki](https://wiki.chatterino.com/Help/)
- [X] I've searched the [issues and pull requests](https://github.com/Chatterino/chatterino2/issues?q=) for similar looking reports

### Describe your issue

I find that the previews are very hard to read with it being so small. Sometimes the preview is slightly bigger than others, but for example the runescape screenshots caught by Scriptorex are way too small. No matter how close I get to my screen I cannot (LUL) make out what is suppose to be portrayed. My display might just be bugged though I can send a screen shot of it 

It seems like the first link preview in #1664 was a _little_ bit bigger, but still not big enough.

### Screenshots

![chatterino link preview issue](https://user-images.githubusercontent.com/97059214/148265551-78fa1e8d-1902-4446-a624-20bb59080f4b.png)

_No response_

### OS and Chatterino Version

Chatterino Nightly + Dankerino patches 2.3.4 (commit 3044f4a5d, ) on Windows 10

**Comment by ALazyMeme** [comment]
This is a suggestion, not an issue. Please carefully read and follow the issue templates - which would've lead you to creating a discussion:
![image](https://user-images.githubusercontent.com/12804673/148039950-c92471a9-93f5-41b9-b4a0-170e9527bab4.png)

**Reply by romeogiggletoes** [reply]
This one is an issue as I cannot read the small lettering or see what the small picture preview is about half of the time without leaning forward.

Edit: for reference #2688 is considered an issue
Edit 2: I was just informed that previously issues also had suggestions before discussions, so first edit is void? Although it's still an 'issue' 🤔

**Reply by ALazyMeme** [reply]
The way this was worded indicates that it's a feature request, not an issue. As per the issue template, your issue should focus on reporting the issue; not suggesting a solution. You should post suggestions in a new comment as opposed to the original post.

![image](https://user-images.githubusercontent.com/12804673/148192897-dd1da642-ed8b-4041-b9c9-8c78a3beee42.png)

**Reply by ALazyMeme** [reply]
Please fix the description accordingly and I will convert back to an issue.

**Reply by romeogiggletoes** [reply]
Ah, I should have read that closer before sending, my mistake. Fixed 👍

**Comment by romeogiggletoes** [comment]
Possible fix: Modify the 'large' option for link previews to be bigger or alternatively make an extra large option. I have shown below how big I thing it should be. 
![Chatterino Link Preview](https://user-images.githubusercontent.com/97059214/148266550-837886bb-aae5-4542-a4d2-70f98db70d3d.png)

Edit: With some help from Karar I learned that you can put in pixels how large you want the preview, however it's way too blurry to be useful. What I was more so referring to was to make the preview bigger while keeping the same image quality. As a side note, I do like that there can be custom values for this and not just the default presets, however as previously stated, anything larger than the 'Large' preset isn't really usable.

![2022-01-05 14_29_31-Chatterino Settings](https://user-images.githubusercontent.com/97059214/148277214-50e6cbb3-e362-4312-b0ba-f83abb2fb102.png)
![2022-01-05 14_30_15-Link Preview for ants 🐜 · Discussion #3462 · Chatterino_chatterino2 — Mozilla F](https://user-images.githubusercontent.com/97059214/148277318-7dc78359-be81-448b-b656-96ae6368d743.png)
